src/eureka/S5_lightcurve_fitting/differentiable_models/StarryModel.py

```diff
@@ -185,8 +195,51 @@ def setup(self):
             planet.theta0 = 180.0
             planet.t0 = temp.t0
 
+            # Pixel sampling setup
+            if 'pixel_ydeg' in self.paramtitles:
+                # Get pixel transform matrix and number of pixels
+                if self.ydeg == 2:
+                    self.oversample = 3
+                else:
+                    self.oversample = 2
+
+                A = planet.map.get_pixel_transforms(
+                    oversample=self.oversample)[3]
+                self.npix = A.shape[1]
+
+                # Set prior to either be log normal, or normal around zero
+                if self.force_positivity:
+                    p = pm.LogNormal("p", mu=np.log(0.3*amp), tau=1.0,
+                                     shape=(self.npix,))
+                else:
+                    p = pm.Normal("p", mu=0.2*amp, sd=0.2*amp,
+                                  shape=(self.npix, ))
```

Comment:
Is a prior of 0.2*fp ± 0.2*fp really broad enough to be minimally informative and be hardcoded?

Reply:
I've changed this to be set by independent "pixel_prior_mean" and "pixel_prior_width" variables in the run parameters file; I didn't use the default free parameter setup as it doesn't include LogNormal priors so this seemed simpler. Using the peak value of the light curve (same as the fp parameter) works fine for both the mean and width parameters. I wouldn't expect a need to use anything different for either parameter, but this gives the option if needed